frontend/src/components/Trial/Trial.tsx

```diff
@@ -114,18 +82,56 @@ const Trial = (props: TrialProps) => {
                     onNext();
                 }
 
+                return;
+            }
+
+            const { form = [] } = feedback_form;
+
+            if (result.type === "time_passed") {
+                form.map((formElement) => (formElement.value = "TIMEOUT"));
+            }
+
+            if (feedback_form.is_skippable) {
+                form.map((formElement => (formElement.value = formElement.value || '')))
             }
+
+            const breakRoundOn = config.break_round_on;
+            const shouldBreakRound = breakRoundOn && checkBreakRound(form.map((formElement) => formElement.value), breakRoundOn);
+            const shouldCallOnNextInOnResult = !shouldBreakRound
```

Comment:
Perhaps we can even lose this variable, as the docstring below actually already explains why a boolean is passed into the `onResult` function.

Reply:
Can't we change this to:

```tsx
const shouldBreakRound = breakRoundConditions && checkBreakRound(form.map((formElement) => formElement.value), breakRoundConditions);
            const shouldCallOnNextInOnResult = !shouldBreakRound

            await onResult(
                {
                    decision_time: getAndStoreDecisionTime(),
                    form,
                    config
                },
                false,
                false // We already call `onNext` _after_ `onResult` as we cannot pass the `shouldBreak` param `onResult`'s `onNext` call.
            );

            onNext(shouldBreakRound);
```